src/token/components/TokenRow.tsx

```diff
@@ -46,21 +23,27 @@ const styles = {
   },
 } as Record<string, CSSProperties>;
 
-export const TokenRow = memo(function TokenRow({ token, amount, onClick }: TokenRowReact) {
+export const TokenRow = memo(function TokenRow({
+  token,
+  amount,
+  onClick,
+  hideImage,
+  hideSymbol,
+}: TokenRowReact) {
   return (
-    <button data-testid="ockTokenRow_Container" style={styles.row} onClick={() => onClick?.(token)}>
-      <span style={styles.left}>
-        {token.image === null ? (
-          <span data-testid="ockTokenRow_PlaceholderImage" style={styles.circle} />
-        ) : (
-          <img data-testid="ockTokenRow_Image" style={styles.image} src={token.image} />
-        )}
-        <span style={styles.column}>
-          <span style={styles.label1}>{token.name}</span>
-          <span style={styles.label2}>{token.symbol}</span>
+    <button
+      data-testid="ockTokenRow_Container"
+      className="ock-tokenrow-button"
+      onClick={() => onClick?.(token)}
+    >
+      <span className="ock-tokenrow-left">
+        {!hideImage && <TokenImage token={token} size={32} />}
```

Comment:
Why someone would prefer hide an image on purpose? 🤔

Reply:
Oh never mind, I see now the point, if you just want to show Symbol and Name. It feels odd, but I guess makes sense.